components/Signout/index.js

```diff
@@ -0,0 +1,54 @@
+import { device } from "@/utils/globalValues";
+import styled from "styled-components";
+import { useSession, signOut } from "next-auth/react";
+import { StyledSettingsButton } from "../StyledComponents/StyledButtons";
+
+export default function Signout() {
+  const { data: session } = useSession();
+
+  if (session?.user?.username) {
+    return (
+      <>
+        <StyledFixedSection>
+          <InlineParagraph>Hello {session?.user?.username}</InlineParagraph>
+          <StyledSettingsButton onClick={() => signOut()}>
+            Sign Out
+          </StyledSettingsButton>
+        </StyledFixedSection>
+        <StyledCircle />
+      </>
+    );
+  }
+}
+
+const InlineParagraph = styled.p`
+  display: inline-block;
+  background-color: transparent;
+  margin-right: 10px;
+`;
```

Comment:
Why not use `<span>`? That's more or less a default inline paragraph. And I'm not sure if `background-color: transparent;` needs to be defined.